nevergrad/optimization/optimizerlib.py

```diff
@@ -797,6 +808,7 @@ class ConfSplitOptimizer(base.ConfiguredOptimizer):
         number of optimizers
     num_vars: optional list of int
         number of variable per optimizer.
+    progressive: optional bool
```

Comment:
please add a definition for this parameter

Reply:
Done, thx.